settings.gradle

```diff
@@ -42,6 +42,7 @@ include 'test-resources-redis'
 include 'test-resources-rabbitmq'
 include 'test-resources-server'
 include 'test-resources-testcontainers'
+include 'test-resources-vault'
```

Comment:
:x: I would use `test-resources-hashicorp-vault`, because there are other kinds of vaults we might support in the future (e.g Oracle Cloud Infrastructure Vault).

Reply:
Agreed 👍